std/typecons.d

```diff
@@ -7093,3 +7093,194 @@ unittest
         ubyte, ubyte, T3, T3,
     );
 }
+
+/**
+ * Get a field or property member function by `ref`. Allows getting fields
+ * via runtime information rather than requiring it at compile-time.
+ *
+ * Params:
+ *     obj = the struct or class to access
+ *     name = the name of the field or property method
+ * Returns:
+ *     The value of the field
+ */
+auto getFeild(Field, Obj)(auto ref Obj obj, string name)
+{
+    enum refObj = !is(Obj == class) && (Obj.sizeof > 16);
+    return rtPropDispatch!(false, Field, false, Obj, refObj)(obj, name);
+}
+
+///
+@safe @nogc nothrow pure unittest
+{
+    struct Foo
+    {
+        int bar = 42;
+    }
+
+    Foo foo;
+    int val = foo.getFeild!int("bar");
+    assert(val == 42);
+}
+
+@safe nothrow pure unittest
+{
+    class A
+    {
+        int bar = 42;
+    }
+
+    auto a = new A();
+    int val = a.getFeild!int("bar");
+    assert(val == 42);
+
+    struct B
+    {
+        auto bar() @property { return 42; }
+    }
+
+    B b;
+    int val2 = b.getFeild!int("bar");
+    assert(val == 42);
+}
+
+/**
+ * Get a field or property member function. Allows getting fields
+ * via runtime information rather than requiring it at compile-time.
+ *
+ * Params:
+ *     obj = the struct or class to access
+ *     name = the name of the field or property method
+ * Returns:
+ *     The value of the field or property method by reference
+ */
+ref Field refGetFeild(Field, Obj)(Obj obj, string name) if (is(Obj == class))
```

Comment:
I see, anyway it seems that GH is malfunctioning right now, The doc builder displays `Pull request SHA mismatch`. Never seen that before.